Лабораторная работа 2/task3.5.2.py

```diff
@@ -0,0 +1,11 @@
+# TODO Напишите функцию find_common_participants
+list_=()
+def find_common_participants(participants_group):
+
+    print(participants_first_group.intersection(participants_second_group))
+
+    return find_common_participants(participants_first_group,participants_second_group)
```

Comment:
Добрый день!
При вызове данной функции можно задать 3 входных значения.
Далее можно создать несколько множеств с уникальными значениями при помощи метода `set()`. При этом указать, что объекты разделяются при помощи разделителя, указанного в переменной `delimiter`.

Затем, при помощи метода `intersection()` выполнить пересечение двух полученных множеств. Таким образом, будет сформировано множество `common_participants`, где будут указаны все повторяющиеся значения в двух множествах `participants_first_group` и `participants_second_group`.

```suggestion
def find_common_participants(participants_first_group, participants_second_group, delimiter =','):

    participants_first_group = set(participants_first_group.split(delimiter))
    participants_second_group = set(participants_second_group.split(delimiter))
    # разделили зяпятыми, переход в множества

    common_participants = participants_first_group.intersection(participants_second_group)
    # пересечение

    common_participants = sorted(list(common_participants)) # список по алфавиту
    return common_participants
```